src/Runner/Runner.php

```diff
@@ -134,8 +134,16 @@ public function fix()
             Tokens::clearCache();
 
             if ($fixInfo) {
-                $name = $this->directory->getRelativePathTo($file);
-                $changed[$name] = $fixInfo;
+                $path = $file->getPathname();
+
+                /* workaround for Symfony/Finder setting doubled slash for root folders in
+                SplFileInfo. See `RecursiveDirectoryIterator::current()`. */
+                if (substr($path, 0, 2) === '//') {
```

Comment:
you provided examples like:
> `soap\\src\client.php`
> ` /srv/home/bpacholek/fail//broken.php`

and then provided code for `//` but not `\\`. probably with tests it will be easier to discuss.